src/redux/storeMock.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable import/no-extraneous-dependencies */
+// import configureStore from 'redux-mock-store';
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable.